### PR TITLE
Adapt GUI

### PR DESCRIPTION
**Pull request by ammarcsj**
Some slight adaptations in the GUI layout

### CODE REVIEW

File: requirements/requirements.txt [hunk 1]
```diff
@@ -5,5 +5,5 @@ dask>=2023.1.0
 numba==0.56.4
 multiprocess==0.70.14
 wget==3.2
-PyYAML==6.0.2
+PyYAML==6.0
```
Comment (mschwoer) [inline requirements/requirements.txt:8]: I think `6.0` has some crucial bug that prevent installation .. was there a reason for reverting from `6.0.2`?

**Review by mschwoer — APPROVED** [review]
LGTM (did not deep dive into the frontend code though)

File: requirements/requirements.txt (hunk 1 above)
Comment (ammarcsj) [reply requirements/requirements.txt:8]: No specific reason, I just adapted when installing new and then using `pip -list`. Will try with the old version

**Comment by ammarcsj** [comment]
thx!

**Pull request merged**